js/repositories.bzl

```diff
@@ -35,7 +39,8 @@ def rules_js_dependencies():
     maybe(
         http_archive,
         name = "aspect_bazel_lib",
-        sha256 = "e834c368f36cb336b5b42cd1dd9cd4b6bafa0ad3ed7f92f54a47e5ab436e4f59",
-        strip_prefix = "bazel-lib-0.3.0",
-        url = "https://github.com/aspect-build/bazel-lib/archive/refs/tags/v0.3.0.tar.gz",
+        sha256 = "9306cb42e19221ea8ec76c05481fd508aefb5b37b3e35123f542977dabaa8d4b",
+        strip_prefix = "bazel-lib-" + versions.bazel_lib,
+        #url = "https://github.com/aspect-build/bazel-lib/archive/refs/tags/v{}.tar.gz".format(versions.bazel_lib),
```

Comment:
Can remove this comment?

Reply:
yeah, we could, as soon as we have the next release of bazel-lib we'll just overwrite this block again anyway